### DISCUSSION TITLE
카메라 회전 반응 영역을 알고 싶습니다.

### DISCUSSION THREAD

**Discussion by zlmrzlo** [naverz/zepeto-studio-kor#1030] [5. Q&A]
캐릭터 주변만 포인트 다운 후 드래그로 회전이 잘되고 점프 패드 위쪽 부분은 아예 반응하지 않고 있는데

반응 영역을 넓이면 될 것 같습니다. 그런데 카메라 회전 반응 영역을 컨트롤할 수 있는 방법을 잘 몰라서

알려주시면 감사하겠습니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 혹시 Game 뷰에서 해상도 설정이 되어 있나요?
해상도 설정을 먼저 기기에 맞게 하신 후 다시 테스트 해 보시겠어요?
아래 이슈 참고 부탁 드립니다.
https://github.com/naverz/zepeto-studio-kor/discussions/735#discussioncomment-2734626

**Reply by zlmrzlo** [reply]
기기 해상도 설정은 1334*750입니다.
스크린샷에 사용된 기기 해상도를 기준으로 해상도를 설정해 놓은 상태였습니다.

그리고 다양한 해상도에서 UI도 테스트를 했었는데
어느 날 한 번 뭔가 이상함을 느꼈는데 점프 패드 위쪽 공간에서는 드래그가 먹히지 않더군요.

아마 이 부분은 영역에서 제외되거나 뭔가가 막고 있는 것 같은데
그걸 못 찾고 있는 상황입니다.

그런데 이러한 문제를 겪었던 사람이 이미 존재했던 것으로 보아
코딩할 때 회전 영역을 따로 설정해 놓은 것은 아닐지 생각되네요.

점프 패드 영역 기준으로 아래쪽이랑 상단쪽 일부는 또 회전이 됩니다.
점프 패드 위쪽에 사각형 영역으로 인식이 안되는 부분이 있는 것 같아요.

**Comment by zlmrzlo** [comment]
Canvas screen space가 터치 화면에 영향을 주는 것으로 확인하였고 이 문제는 해결했습니다.
